.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+# CircleCI configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0  # Enables Windows executors
+
+jobs:
+  
+  linux-python-38: &linux-template
+    docker:
+      - image: circleci/python:3.8
+
+    environment:
+      PYTHON: python
+
+    working_directory: ~/repo
+
+    steps:
+
+      - checkout
+      - run:
+          name: install dependencies
+          command: |
+            export PATH=$PATH:$HOME/.local/bin
+            ${PYTHON} -m pip install --progress-bar off -U virtualenv --user
+            mkdir -p ./venv
+            virtualenv ./venv --clear
+            . venv/bin/activate
+            if [[ "$CIRCLE_JOB" != *"pypy"* ]]; then
+              sudo apt update -qq && sudo apt install -y -qq libhdf5-dev
+            fi
+            ${PYTHON} -m pip install --progress-bar off -U pip==18
```

Comment:
@bdice  One question, is there a specific reason for using pip version = 18?

Reply:
No -- we should remove the version requirement.

Reply:
It's usually a good idea to pin as much as possible for these workflows so that failures are more likely stemming from issues that we create as opposed to changes in the ecosystem. It's probably fine to remove, but pinning is not completely pointless.

Reply:
I would be fine with updating this pinning to `pip==20.0.2`